backend/src/test/java/org/partypets/backend/controller/PartyControllerTest.java

```diff
@@ -73,6 +73,31 @@ void expectNewPartyInList_whenPostingParty() throws Exception {
                 //Then
                 .andExpect(MockMvcResultMatchers.content().json(expected)).andExpect(MockMvcResultMatchers.status().isOk());
     }
+
+
+    @Test
+    @DirtiesContext
+    void expectParty_whenGettingByID() throws Exception {
+        //Given
+        Party newParty = new Party("abc", new Date(), "Home", "Dog-Bday");
+        this.partyRepo.setParties(List.of(newParty));
+        String expected = """
+                   
+                        {
+                            "id": "abc",
+                            "location": "Home",
+                            "theme": "Dog-Bday"
+                         }
+                    
+                """;
+
+
+        //When
+        mockMvc.perform(MockMvcRequestBuilders.get("/api/parties/abc"))
+
+                //Then
```

Comment:
Ben says this could be better formatted ;)
```suggestion
     //Then
```

Reply:
Ok will do, but intelliJ autoformats this part by indenting the comment.
Can you approve the PR?